README.md

```diff
@@ -0,0 +1,72 @@
+# elasticache
+
+Provides an ElastiCache Replication Group resource.
+
+Example Usage
+----------------
+
+```hcl
+module "elasticache" {
+  source = "git@github.com:techservicesillinois/terraform-aws-elasticache"
+
+  cluster_name       = "service-name"
+
+  network_configuration {
+    vpc  = "name-of-vpc"
+    tier = "public"
+    security_group_names = "service-redis-clients"
+  }
+}
+```
+
+Argument Reference
+-----------------
+
+The following arguments are supported:
+
+* `cluster_name` - (Required) The name or identifier of the Replication group.  
+
+* `security_group_ids` - (Required) One or more Amazon VPC security groups associated with this replication group. Use this parameter only when you are creating a replication group in an Amazon Virtual Private Cloud
+
+* `redis_clusters` - (Required) The number of cache clusters (primary and replicas) this replication group will have. If Multi-AZ is enabled, the value of this parameter must be at least 2. Updates will occur before other modifications. Defaults to `2`.
```

Comment:
Does it make sense for the default to be 2 cluster nodes? What about the use case where a single redis node suffices for an application?

Reply:
Will explicitly set "redis_clusters" to 2 in configuration.